mender-deb-package

```diff
@@ -240,7 +230,11 @@ EOF
     rm -f debian/patches/series
   fi
 
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6ED0E7B82643E131 2> /dev/null
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0E98404D386FA1D9 2> /dev/null
+
   apt-get update
+  apt-get -y install cmake
```

Comment:
(this is good to stay here, by the way - my above comment is only for the `apt-key` commands.